data/fixtures/scopes/python/name.argument.actual.iteration.scope

```diff
@@ -0,0 +1,16 @@
+aaa(bbb=0, ccc=1)
```

Comment:
Names come from compiler history -- an "actual" parameter is where it is passed, a formal parameter is where it is defined in the function definition (if there is one)